### PR TITLE
Add Aevo Derivatives exchange

### PR DESCRIPTION
**Pull request by germanzelentsov**
### Pull request checklist

- [x] Did you bump the project version?
- [ ] Did you add a description to the Pull Request?
- [ ] Did you add new tests?
- [ ] Did you add reviewers?

### CODE REVIEW

File: src/Aevo/Aevo.jl [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    return print(io, "code = ", "\"", e.code, "\"", ", ", "msg = ", "\"", e.msg, "\"")
+end
+
+function CryptoAPIs.request_sign!(::AevoClient, query::Q, ::String)::Q where {Q<:AevoPublicQuery}
+    return query
+end
+
+function CryptoAPIs.request_sign!(::AevoClient, query::Q, ::String)::Q where {Q<:AevoAccessQuery}
+    return query
+end
```
Comment (yshadrin) [inline src/Aevo/Aevo.jl:81]: it can be combined into one for an abstract type AevoCommonQuery

File: src/Aevo/Aevo.jl [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    return Pair{String,String}[
+        "accept" => "application/json",
+    ]
+end
+
+function CryptoAPIs.request_headers(client::AevoClient, ::AevoAccessQuery)::Vector{Pair{String,String}}
+    return Pair{String,String}[
+        "accept" => "application/json",
+    ]
+end
```
Comment (yshadrin) [inline src/Aevo/Aevo.jl:101]: you can also combine

File: src/Aevo/Derivatives/API/FundingRate.jl [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    mark_price::Float64
+end 
+
+struct FundingRateData <: AevoData
+    funding_history::Vector{FundingHistory}
+end
+
+"""
+    product_stats(client::AevoClient, query::FundingRateQuery)
+    product_stats(client::AevoClient = Aevo.Derivatives.public_client; kw...)
```
Comment (yshadrin) [inline src/Aevo/Derivatives/API/FundingRate.jl:35]:
```suggestion
    funding_rate(client::AevoClient, query::FundingRateQuery)
    funding_rate(client::AevoClient = Aevo.Derivatives.public_client; kw...)
```

File: src/Aevo/Aevo.jl [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    AevoAPIError{T} <: AbstractAPIsError
+
+Exception thrown when an API method fails with code `T`.
+
+## Required fields
+- `code::Int64`: Error code.
+
+## Optional fields
+- `type::String`: Type of error.
+- `msg::String`: Error message.
```
Comment (yshadrin) [inline src/Aevo/Aevo.jl:57]: the description does not match the structure

File: src/Aevo/Utils.jl [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Aevo/Utils
+
+function Serde.deser(::Type{<:AevoData}, ::Type{<:Maybe{NanoDate}}, x::String)::NanoDate
+    return unixnanos2nanodate(parse(Int64, x))
+end
+
+function Serde.SerQuery.ser_type(::Type{<:AevoCommonQuery}, x::D)::Int64 where {D<:DateTime}
+    return round(Int64, 1000 * datetime2unix(x))
```
Comment (yshadrin) [inline src/Aevo/Utils.jl:8]:
```suggestion
    return round(Int64, 1000000 * datetime2unix(x))
```

File: Project.toml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 name = "CryptoAPIs"
 uuid = "5e3d4798-c815-4641-85e1-deed530626d3"
-version = "0.24.1"
+version = "0.26.0"
```
Comment (yshadrin) [inline Project.toml:3]:
```suggestion
version = "0.25.0"
```

File: examples/Aevo/Futures.jl [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Aevo/Futures
+# https://api-docs.aevo.xyz/reference/overview
+
+using Dates
+using CryptoAPIs
+using CryptoAPIs.Aevo
+
+
```
Comment (yshadrin) [inline examples/Aevo/Futures.jl:8]: double enter

File: src/Aevo/Futures/API/FundingRate.jl [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+      "timestamp":"2024-07-08T14:00:00",
+      "funding_rate":1.0e-5,
+      "mark_price":3037.555603
+    },
+    {
+      "instrument_name":"ETH-PERP",
+      "timestamp":"2024-07-08T13:00:00",
+      "funding_rate":2.0e-6,
+      "mark_price":3049.066838
+    }
```
Comment (yshadrin) [inline src/Aevo/Futures/API/FundingRate.jl:118]: I think that this section can be replaced with `...`

**Review by yshadrin — APPROVED** [review]

**Review by gryumov — APPROVED** [review]

**Pull request merged**